ui/src/app/views/workspace-share/component.html

```diff
@@ -32,6 +32,9 @@ <h3 class="modal-title-text">Share {{workspace.name}}</h3>
         <div *ngIf="userNotFound" class="error-message">
           User {{toShare}} not found.
         </div>
+        <div *ngIf="roleNotSelected" class="error-message">
```

Comment:
Does this appear immediately, or only after the user clicks on "Add"?

Reply:
Only after they click add.